packages/mui-material-next/src/SnackbarContent/SnackbarContent.types.ts

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
 import { SxProps } from '@mui/system';
```

Comment:
```suggestion
import { SxProps } from '../styles/Theme.types';
```

Reply:
BTW this one doesn't need the `Theme` parameter to be passed manually: https://github.com/mui/material-ui/blob/master/packages/mui-material-next/src/Chip/Chip.types.ts#L78

Reply:
With this change, I should `Omit` the `sx` props from `Paper` as follows:
`export interface SnackbarContentProps extends Omit<StandardProps<PaperProps, 'children'>, 'sx'> `
Is it ok?

Reply:
Yes ~ it makes sense

Reply:
Considering that 'sx' has been passed to `Paper` and that `mui-material` is used for Paper, we are required to use the Paper 'sx' type until Paper is migrated to material-next.
I have added a TODO for this section.